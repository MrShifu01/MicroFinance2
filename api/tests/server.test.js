const request = require('supertest');
const express = require('express');

// Create an Express app
const app = express();

// Define your route handler
app.get('/test', (req, res) => {
  res.json({ message: 'API is working!' });
});

describe('Route Handler Snapshot Test', () => {
  it('should match the snapshot', async () => {
    const response = await request(app).get('/test');
    expect(response.body).toMatchSnapshot();
  });
});
