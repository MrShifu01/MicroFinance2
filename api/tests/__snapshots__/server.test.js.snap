// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Route Handler Snapshot Test should match the snapshot 1`] = `
{
  "message": "API is working!",
}
`;
